src/components/Button/Button.js

```diff
@@ -97,6 +85,44 @@ Button.propTypes = {
    * adds optional value tag to the button
    */
   value: PropTypes.string,
+  /**
+   * Button flavor: string or strings in array. Flavors: neutral,
```

Comment:
This is something people will stumble over: Do I use a string or an array? Can I concat multiple flavors in one string? E.g. `icon icon-border`. If I see a string-flavor, I would assume that that is the case. This is a bit counter-intuitive

Reply:
Still not sure how to handle this: I could only allow arrays (even for single flavors), or allow concatenated strings as well as single strings and arrays of strings. The problem with concatenated strings is that checking each flavor string for validity with PropTypes is impossible.

Reply:
Not impossible but would require a custom propType. fwiw, I think arrays are the way to go here (in combination with consolidated props like `size`)